### DISCUSSION TITLE
Multi-Call does not work when first call is outbound.

### DISCUSSION THREAD

**Discussion by stjohnson44** [twilio-professional-services/flex-project-template#594] [General]
When the multi-call feature is enabled, if an agent is on a manual outbound call and receives a second, inbound call, the plugin does not work. The experience defaults to the popup that asks to either accept the new call and end the existing call, or reject the incoming call and stay on the existing call.

Is this scenario handled/known?

**Comment by dremin** [comment]
Hi @stjohnson44! The multi-call feature was tested for inbound use cases only, so it doesn't surprise me that it isn't quite right with outbound in the picture.

Thanks for letting us know about this issue!
